### PR TITLE
[PT] Portuguese translation of the second week of the course

### PR DESCRIPTION
**Pull request by lnncrs**
Notes about the translation

I maintained the English blocks in the source as the other versions but sliced them into bigger blocks by topic, I think it facilitates the translation.

I always used the English material as the primary source, in sentences where the translation to Portuguese didn't fully express the final meaning of the source material, I relied on the Spanish version (a version with more contributors and a mature version I think) just for that word to decide to translate or not.

For an example of a not translated term, in the 02-1 after the "Nash Equilibria" the acronym "GAN" (or generative adversarial network) is not translated in Spanish and I didn't translate either, because if you look up this acronym on the Portuguese Google, you will not find the correct explanation about the term, it's not even the same field, in all cases like that, I put the word or acronym in italic (even if it is not in italic on the Spanish translation), so we not got non-translated terms outside italic to easily find and recognize them. Other examples are "RL", "SGD" or "one-hot" which terms are more easily recognizable in Portuguese by the English acronym, but all of them have the Portuguese term right after on parenthesis. The terms softmax and log softmax are exceptions as are many cases in the 02-2 text and are easily recognizable (and used in this form) in Portuguese. The term "Dropout" has no term in Portuguese close or better to the understanding of this meaning (as "drop + out" is common and simple words) and so I maintained the term untouched.

The term "Actor-Critic Methods" was translated in Spanish, but I didn't find much of the "Actor-Critic Methods" translations, except for one academic paper in Portuguese with a similar term for the same technique (https://tedebc.ufma.br/jspui/bitstream/tede/1879/2/Patricia%20Helena.pdf), and so I included this term in the translation, but in parenthesis after the English term. If the reader wants to get deeper into this technique, he will find at least this (or related) academic paper in Portuguese.

As I used the English material as a primary source, there are cases when a term is not translated in Spanish translation but I translated it to Portuguese when the term has a Portuguese mathematical (or universal) recognizable form, the better example is "negative log-likelihood loss" which I understood as a "negative logarithmic probabilistic function used as a loss function" in this case, I translated and included the source English form in the parenthesis.

The term "logits" was not translated either because I found multiple explanations (as in https://stackoverflow.com/questions/41455101/what-is-the-meaning-of-the-word-logits-in-tensorflow) for this term into the same field, and as in the Spanish version I maintained de original term.

### CODE REVIEW

**Review by Atcold — APPROVED** [review]
Great, thanks!
I've updated `_config.yml` for you.
